site/docs/components/progress-stepper.mdx

```diff
@@ -9,11 +9,10 @@ needToKnow:
 - Progress steppers may be gated or ungated.
 - A step may show validation feedback.
 - Aim for 3–4 steps.
-demoStoryId: deprecated-vertical-progress-step--completed-step
 health:
   designed: true
   documented: true
-  implemented: true
+  implemented: false
   latestDesign: false
   allVariants: false
   responsive: false
```

Comment:
Probs not the right time to do it but could `deprecated: true` be useful?

Reply:
Actually ignore more, for some reason i thought this component might be deprecated and thats why you were deleteing it

Reply:
The component _idea_ itself isn't deprecated though. There's designs for a horizontal one in the UI Kit that we'll want to build at some point, and possibly a rebuild of this vertical one as well.